src/Text/Editor.hs

```diff
@@ -104,6 +106,31 @@ runUserEditorWithTemplate templ =
   \theEditor ->
     runSpecificEditor theEditor templ mempty
 
+--- |This is the function you should use.
+--- 
+--- 'bracketConduit' takes a 'Producer', to produce the contents of the
+--- original file, and a 'Consumer' to consume them, and returns the
+--- result of the consumer.
+--- 
+--- If you don't know how to use conduits, see the
+--- <http://www.stackage.org/package/conduit documentation> for the
+--- conduit package.
+--- 
+--- If you really don't want to use conduits, you can use the strict I/O
+--- functions
+bracketConduit :: Template 
+               -> Producer (ResourceT IO) ByteString
+               -> Consumer ByteString (ResourceT IO) b
+               -> ResourceT IO b
+bracketConduit template source consumer =
```

Comment:
Could you rewrite this section using the `connect` and `fuse` functions from conduit. I can't stand infix operators.

Instead of lift `liftIO $` before every IO operation, you could just write `liftIO $ do ...`, and you wouldn't have to run `liftIO` on every operation inside the `do`-block.